### PR TITLE
Add requirement for letting people in

### PR DESCRIPTION
**Pull request by halkeye**
@keywordnew @Qard 

I would like help wording this. The venue this weekend was nice, but I had to both run the event, and go down and shuttle people up the elevator. Even make a sign myself, but we had no tape so it sorta got tucked into a plant in the lobby. Luckily I handed this off to nigel as we got started, but it should be a requirement.

### CODE REVIEW

File: HOSTING.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ Are you interested in hosting NodeSchool Vancouver? We have a minimum set of req
   * Can be split into a couple rooms because we tend to split into smaller groups
 * Wifi for 20+ people
 * Power
+* A plan for letting people into space (Since event happens on weekends). This should include:
```
Comment (keywordnew) [inline HOSTING.md]:
```suggestion
* A plan for letting people into space. Most commercial offices have restricted access on weekends. This plan should include:

```

File: HOSTING.md [hunk 2]
```diff
@@ -8,6 +8,9 @@ Are you interested in hosting NodeSchool Vancouver? We have a minimum set of req
   * Can be split into a couple rooms because we tend to split into smaller groups
 * Wifi for 20+ people
 * Power
+* A plan for letting people into space (Since event happens on weekends). This should include:
+  * A sign if needed.
```
Comment (keywordnew) [inline HOSTING.md]:
```suggestion
  * A sign: getting from the main entrance of a building to a space is more difficult to people who have not been there before. This applies to most attendees. Unless the path from main entrance to the space has no possible diversions, you need signs.
```

**Review by keywordnew — COMMENTED** [review]
LGTM.

Sorry to not have documented this earlier, @halkeye. It's a common pain.

**Comment by Qard** [comment]
I would also add a note to specifically communicate if the host will provide a _person_ that can help with that too, or if the nodeschool team will be on their own.

**Comment by halkeye** [comment]
Yea absolutely. @keywordnew I can take a stab at I, but would love you wording if you get a chance.

Though honestly I also feel like it should be part of hosting requirement that they provide someone so I'm not sure we want to provide the option not to

**Comment by keywordnew** [comment]
@halkeye Ok! For now, I committed the earlier suggestions. Hope that's ok, feel free to change.

**Comment by halkeye** [comment]
Yea let's merge and new pr for the new bits!

**Pull request merged**